### DISCUSSION TITLE
add more specifics to the installation instruction

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/GenerativeModels#442] [Q&A] [answered]
Hi there!
Really want to give this exciting repo a try!
In the [installation instruction](https://github.com/Project-MONAI/GenerativeModels#installation), could you specify any packages (like the version of python) need to be pre-installed before `python setup.py install`? I guess lots of users will want to install it in a conda env.

**Comment by marksgraham** [comment, marked as answer]
Hi there,

As the package is built on MONAI it has the same [python requirements](https://docs.monai.io/en/latest/installation.html#installation-guide): python >=3.8. Everything else needed should be installed when you run setup-py, though I should note you can also install through pip with`pip install monai-generative`. I'll look into updating the instructions!

**Reply by hw-ju** [reply]
Hi @marksgraham, 
Thanks a lot for your prompt and helpful response! BTW, thanks for your great presentation at the GTC23 session, very informative :).

**Reply by marksgraham** [reply]
glad you liked the presentation :)
